### PR TITLE
feat(knockout): overview page

### LINKED ISSUE #14: Create knockout overview component
https://codepen.io/RobbertWolfs/pen/YeoLvY?editors=1100

### PR DESCRIPTION
**Pull request by RobbertWolfs**
fix #14

### CODE REVIEW

File: src/pages/knockout.vue [hunk 1]
```diff
@@ -17,19 +22,77 @@ import { getRoundName } from '../helpers/knockout'
...
       rounds: 'knockout/rounds',
+      activeRound: 'knockout/round',
+      activeRoundId: 'knockout/activeRoundId',
+      activeRoundState: 'knockout/activeRoundState',
+      winner: 'knockout/winner',
+      tournamentComplete: 'knockout/complete'
     }),
 
     totalRoundsClass() {
+      console.log(this.rounds);
```
Comment (WartClaes) [inline src/pages/knockout.vue]: log

File: src/store/modules/knockout/actions.js [hunk 2]
```diff
@@ -28,6 +28,99 @@ export default {
...
+  generateRound({ commit, getters }) {
+    const tournament = getters.tournament;
+    const teams = tournament.teams;
+    const roundIndex = getters.activeRoundId;
+    const rounds = getters.rounds;
+
+    const winningTeams = [];
+    tournament.knockout.rounds[roundIndex].forEach(match => {
+      switch(match.winner) {
+        case 1:
```
Comment (WartClaes) [inline src/store/modules/knockout/actions.js]: config -> matchWinner

File: src/store/modules/knockout/actions.js [hunk 3]
```diff
@@ -28,6 +28,99 @@ export default {
...
+      tournament: getters.tournament
+    });
+  });
+
+  setWinner({commit, getters}) {
+    const tournament = getters.tournament;
+    const rounds = tournament.knockout.rounds;
+    const final = rounds[rounds.length - 1][0];
+    const winner = final.winner;
+    console.log('fina;', final, winner);
```
Comment (WartClaes) [inline src/store/modules/knockout/actions.js]: log

File: src/store/modules/knockout/actions.js [hunk 4]
```diff
@@ -28,6 +28,99 @@ export default {
...
+    const winningTeamId = winner === 1 ? final.home.id : final.away.id;
+    console.log('fina;', winningTeamId);
```
Comment (WartClaes) [inline src/store/modules/knockout/actions.js]: log

File: src/pages/knockout.vue (hunk 1 above)
Comment (RobbertWolfs) [reply src/pages/knockout.vue]: removed

File: src/store/modules/knockout/actions.js (hunk 2 above)
Comment (RobbertWolfs) [reply src/store/modules/knockout/actions.js]: fixed

File: src/store/modules/knockout/actions.js (hunk 3 above)
Comment (RobbertWolfs) [reply src/store/modules/knockout/actions.js]: removed

**Pull request merged**